force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.js

```diff
@@ -0,0 +1,74 @@
+import { LightningElement, track } from 'lwc';
+const devFundWeight = 0.23;
+const processAutoWeight = 0.30;
+const userIntWeight = 0.25;
+const testDebugWeight = 0.22;
+const passingScore = 68;
+
+export default class PlatformDevCertCalculator extends LightningElement {
+
+  devFundamentalScore = 50;
+  processAutomationScore = 50;
+  userInterfaceScore = 50;
+  testingScore = 50;
+  certificationScore = 90;
+  numberOfQuestions = 60;
+  showResources = false;
+  showGoodJob = false;
+  currentHistoryId = 0;
+
+  @track attemptHistory = [];
+
+  calculateScore(){
```

Comment:
Try adding a validation to this method that shows up if the final score is too high or low